vulnerabilities/severity_systems.py

```diff
@@ -0,0 +1,54 @@
+import dataclasses
+
+
+@dataclasses.dataclass
+class ScoringSystem:
+
+    # a short identifier for the scoring system.
+    identifier: str
+    # a name which represents the scoring system such as `RedHat bug severity`.
+    # This is for human understanding
+    name: str
+    # a url to documentation about that sscoring system
+    url: str
+    # notes about that scoring system
+    notes: str = ""
+
+    def as_score(self, value):
+        """
+        Return a normalized numeric score for this scoring system  given a raw
+        value. For instance this can be used to convert a CVSS vector to a base
+        score.
+        """
+        raise NotImplementedError
+
+
+scoring_systems = {
+    "cvssv2": ScoringSystem(
+        identifier="cvssv2",
+        name="CVSSv2",
+        url="https://www.first.org/cvss/v2/",
+        notes="We store the vector as value and compute scores from that.",
```

Comment:
Based on the new design suggestion here, we will store the vector and values each as different `ScoringSystem`